i18n(it): translate `guides/customization` (#654)
withastro/starlight: docs/src/content/docs/it/guides/customization.mdx

   ```js
   // astro.config.mjs
   import { defineConfig } from 'astro/config';
   import starlight from '@astrojs/starlight';

   export default defineConfig({
   	integrations: [
   		starlight({
   			title: 'Documentazione con il mio logo',
   			logo: {
   				src: './src/assets/my-logo.svg',
   			},
   		}),
   	],
   });
   ```

Per impostazione predefinita, il logo verrà visualizzato accanto al `title` del tuo sito.
Se l'immagine del tuo logo include già il titolo del sito, puoi nascondere visivamente il testo del titolo impostando l'opzione `replacesTitle`.
Il testo del `title` verrà comunque incluso per gli screen reader in modo che l'intestazione rimanga accessibile.

```js
starlight({
  title: 'Documentazione con il mio logo',
  logo: {
    src: './src/assets/my-logo.svg',
    replacesTitle: true,
  },
}),
```

### Varianti del logo chiaro e scuro

Puoi visualizzare diverse versioni del tuo logo in modalità chiara e scura.

1. Aggiungi un file immagine per ciascuna variante a `src/assets/`:

   <FileTree>

   - src/
     - assets/
       - **light-logo.svg**
       - **dark-logo.svg**
     - content/
   - astro.config.mjs

   </FileTree>

> 2. Aggiungi il percorso alle varianti del tuo logo come opzioni "light" e "dark" invece di "src" in "astro.config.mjs":

Reviewer: Qua servono ` invece di ":

```suggestion
2. Aggiungi il percorso alle varianti del tuo logo come opzioni `light` e `dark` invece di `src` in `astro.config.mjs`:
```

Author: Yep, missed them.

   ```js
   starlight({
     title: 'Documentazione con il mio logo',
     logo: {
       light: './src/assets/light-logo.svg',
       dark: './src/assets/dark-logo.svg',
     },
   }),
   ```

> 

Reviewer: Qua abbiamo aggiunto una nuova sezione in #637:

````md
## Enable sitemap

Starlight has built-in support for generating a sitemap. Enable sitemap generation by setting your URL as `site` in `astro.config.mjs`:

```js
// astro.config.mjs

export default defineConfig({
  site: 'https://stargazers.club/',
  integrations: [
    starlight({ title: 'Site with sitemap' }),
  ],
});
```
````

Potresti aggiungere questo nuovo paragrafo a questo PR?

## Layout della pagina

Per impostazione predefinita, le pagine Starlight utilizzano un layout con una barra laterale di navigazione globale e un sommario che mostra le intestazioni della pagina corrente.

Puoi applicare un layout di pagina più ampio senza barre laterali impostando [`template: splash`](/it/reference/frontmatter/#template) nel frontmatter di una pagina.
Funziona particolarmente bene per le pagine di destinazione e puoi vederlo in azione sulla [home page di questo sito](/it/).

```md
---
# src/content/docs/index.md

> tititletolo: La mia pagina di destinazione
> template: spruzzo

Reviewer: 😅 
```suggestion
title: La mia pagina di destinazione
template: splash
```

Author: Oh boy, sorry.

---
```

> ## Sommario

Reviewer: Ma in italiano si dice “sommario” oppure “indice dei contenuti” (o anche solo “indice”)? Guardando [Wikipedia](https://it.wikipedia.org/wiki/Indice_(editoria)), dice “chiamato impropriamente anche sommario” ma forse dipende dal contesto.

Author: Viene usato liberamente, probabilmente Wikipedia ha ragione sul fatto che "indice dei contenuti" sia quello corretto formalmente. Diciamo che "sommario" è più un riassunto, ragionandoci ora. Correggo.

Reviewer: Grazie!

```js
starlight({
  title: 'Documentazione con il mio logo',
  logo: {
    src: './src/assets/my-logo.svg',
    replacesTitle: true,
  },
}),
```

### Varianti del logo chiaro e scuro

Puoi visualizzare diverse versioni del tuo logo in modalità chiara e scura.

1. Aggiungi un file immagine per ciascuna variante a `src/assets/`:

   <FileTree>

   - src/
     - assets/
       - **light-logo.svg**
       - **dark-logo.svg**
     - content/
   - astro.config.mjs

   </FileTree>

2. Aggiungi il percorso alle varianti del tuo logo come opzioni `light` e `dark` invece di `src` in `astro.config.mjs`:

   ```js
   starlight({
     title: 'Documentazione con il mio logo',
     logo: {
       light: './src/assets/light-logo.svg',
       dark: './src/assets/dark-logo.svg',
     },
   }),
   ```

## Enable sitemap

Starlight ha il supporto integrato per la generazione di una mappa del sito. Abilita la generazione della mappa del sito impostando il tuo URL come `site` in `astro.config.mjs`:

```js
// astro.config.mjs

export default defineConfig({
	site: 'https://stargazers.club/',
	integrations: [starlight({ title: 'Sito con mappa del sito' })],
});
```

## Layout della pagina

> Per impostazione predefinita, le pagine Starlight utilizzano un layout con una barra laterale di navigazione globale e un sommario che mostra le intestazioni della pagina corrente.

Reviewer: ```suggestion
Per impostazione predefinita, le pagine Starlight utilizzano un layout con una barra laterale di navigazione globale e un indice dei contenuti che mostra le intestazioni della pagina corrente.
```

Puoi applicare un layout di pagina più ampio senza barre laterali impostando [`template: splash`](/it/reference/frontmatter/#template) nel frontmatter di una pagina.
Funziona particolarmente bene per le pagine di destinazione e puoi vederlo in azione sulla [home page di questo sito](/it/).

```md
---
# src/content/docs/index.md

title: La mia pagina di destinazione
template: splash
---
```

## Indice dei contenuti

> Starlight visualizza un sommario su ogni pagina per consentire ai lettori di passare più facilmente all'intestazione che stanno cercando.
> Puoi personalizzare, o addirittura disabilitare, il sommario a livello globale nell'integrazione Starlight o pagina per pagina in Frontmatter.
> 
> Per impostazione predefinita, le intestazioni `<h2>` e `<h3>` sono incluse nel sommario. Modifica i livelli delle intestazioni da includere in tutto il sito utilizzando le opzioni `minHeadingLevel` e `maxHeadingLevel` nel tuo [global `tableOfContents`](/it/reference/configuration/#tableofcontents). Sostituisci queste impostazioni predefinite su una singola pagina aggiungendo le proprietà [frontmatter `tableOfContents`](/it/reference/frontmatter/#tableofcontents) corrispondenti:

Reviewer: ```suggestion
Starlight visualizza un indice dei contenuti su ogni pagina per consentire ai lettori di passare più facilmente all'intestazione che stanno cercando.
Puoi personalizzare, o addirittura disabilitare, l'indice dei contenuti a livello globale nell'integrazione Starlight o pagina per pagina in Frontmatter.

Per impostazione predefinita, le intestazioni `<h2>` e `<h3>` sono incluse nell'indice dei contenuti. Modifica i livelli delle intestazioni da includere in tutto il sito utilizzando le opzioni `minHeadingLevel` e `maxHeadingLevel` nel tuo [global `tableOfContents`](/it/reference/configuration/#tableofcontents). Sostituisci queste impostazioni predefinite su una singola pagina aggiungendo le proprietà [frontmatter `tableOfContents`](/it/reference/frontmatter/#tableofcontents) corrispondenti:
```

<Tabs>
  <TabItem label="Frontmatter">

```md
---
# src/content/docs/example.md

> title: pagina con solo H2 nel sommario

Reviewer: ```suggestion
title: Pagina con solo H2 nell'indice dei contenuti
```

summary:
  minHeadingLevel: 2
  maxHeadingLevel: 2
---
```

  </TabItem>
  <TabItem label="Configurazione globale">

```js
// astro.config.mjs

defineConfig({
	integrations: [
		starlight({
			title: '',
			tableOfContents: { minHeadingLevel: 2, maxHeadingLevel: 2 },
		}),
	],
});
```

   </TabItem>
</Tabs>

> Disabilita completamente il sommario impostando l'opzione `tableOfContents` su `false`:

Reviewer: ```suggestion
Disabilita completamente l'indice dei contenuti impostando l'opzione `tableOfContents` su `false`:
```

<Tabs>
   <TabItem label="Frontmatter">

```md
---
# src/content/docs/esempio.md

> title: pagina senza sommario

Reviewer: ```suggestion
title: Pagina senza indice dei contenuti
```

> tableOfContents: falso

Reviewer: Oh, non avevo visto questo prima!

```suggestion
tableOfContents: false
```

---
```

   </TabItem>
   <TabItem label="Configurazione globale">

```js
// astro.config.mjs

defineConfig({
	integrations: [
		starlight({

> 			title: 'Documenti con sommario disabilitato a livello globale',

Reviewer: ```suggestion
			title: 'Documentazione con indice dei contenuti disabilitato a livello globale',
```
